modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/SubsystemControlCell.scala

```diff
@@ -42,9 +47,11 @@ object SubsystemControlCell {
     id:     Observation.Id,
     stepId: StepId,
     r:      Resource
-  )(e:      ReactEvent): Callback =
+  )(e:      ReactEvent): Callback = {
+    if (r === Instrument.Gpi) logger.info(s"*** SubsystemControlCell requestResourceCall: $r")
```

Comment:
If you need this you can do `Callback.log`

Reply:
👍 Thanks!